app/views/peer_reviews/_peer_reviews_new_manager.js.jsx.erb

```diff
@@ -0,0 +1,414 @@
+<script type='text/jsx'>
+  /** @jsx React.DOM */
+
+  // The main manager that controls all the components.
+  var PeerReviewsManager = React.createClass({
+    getInitialState: function() {
+      return {
+        students: [],
```

Comment:
The state names need to change. Should be "reviewer_groups" and "reviewee_groups" (+ selected variants), rather than students and groups.